planner/core/src/main/java/eu/seaclouds/platform/planner/core/Planner.java

```diff
@@ -296,32 +309,31 @@ public SLAInfo() {}
 
     }
 
-    class OfferJson {
+    public static class Offering {
```

Comment:
Those are two different domains, and that class in the discoverer is changed (and will be PR soon)